test/javascripts/acceptance/sort-event-topics-test.js

```diff
@@ -0,0 +1,24 @@
+import { acceptance, exists } from "discourse/tests/helpers/qunit-helpers";
+import { test } from "qunit";
+import { visit } from "@ember/test-helpers";
+import { cloneJSON } from "discourse-common/lib/object";
+import CategoryFixtures from "discourse/tests/fixtures/category-fixtures";
+
+acceptance("Calendar - Disable sorting headers", function (needs) {
+  needs.user();
+  needs.pretender((server, helper) => {
+    const cardResponse = cloneJSON(CategoryFixtures["/c/1/show.json"]);
```

Comment:
The _smallest_ of suggestions: `cardResponse` probably should be `categoryResponse` or similar.

Reply:
You're right. Thanks.